sklearn/linear_model/ridge.py

```diff
@@ -854,7 +859,11 @@ def _errors_and_values_helper(self, alpha, y, v, Q, QT_y):
         -----
         We don't construct matrix G, instead compute action on y & diagonal.
         """
-        w = 1.0 / (v + alpha)
+        w = 1. / (v + alpha)
+        dummy_column = np.var(Q, 0) < 1.e-12
```

Comment:
I think that I would call this "constant_column" rather than "dummy_column", no?